apps/scraper-runner/src/main.ts

```diff
@@ -60,6 +61,7 @@ async function findCorrelatedParagraphs(
       (paragraph: ParsedArticleParagraph) => {
         return {
           head: article.head,
+          backsUpClaim: ArticleParagraphBacksUpClaim.notApplicable,
```

Comment:
is this temporarily hardcoded to notApplicable and will change in the future?

Reply:
Its currently the default which can later get changed within the saved json files. I'll add that as a comment